pkg/syncer/syncer.go

```diff
@@ -134,9 +141,18 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 
 	upstreamConfig := rest.CopyConfig(cfg.UpstreamConfig)
 	upstreamConfig.Host = syncerVirtualWorkspaceURL
-	rest.AddUserAgent(upstreamConfig, "kcp#spec-syncer/"+kcpVersion)
+	rest.AddUserAgent(upstreamConfig, "kcp#syncing/"+kcpVersion)
```

Comment:
don't think that's a valid user agent.

Reply:
It seems that the way user agents are set in the Syncer should be reworked globally to bring back both inner consistency, and also consistency with other KCP components.

Dedicated issue created: https://github.com/kcp-dev/kcp/issues/2581